packages/account/src/Sections/Security/AccountClosed/account-closed.tsx

```diff
@@ -40,6 +45,6 @@ const AccountClosed = ({ logout }) => {
     );
 };
 
-export default connect(({ client }) => ({
+export default connect(({ client }: RootStore) => ({
```

Comment:
I think RootStore has a type `TRootStore`. Can u check if that can be used here?